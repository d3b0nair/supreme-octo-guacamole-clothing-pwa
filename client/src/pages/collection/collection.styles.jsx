import styled from "styled-components";

export const CollectionPageContainer = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
`;

export const CollectionPageTitle = styled.h2`
  font-size: 38px;
  margin: 0 auto 30px;
`;

export const CollectionPageItems = styled.div`
  display: grid;
  grid-template-columns: 1fr 1fr 1fr 1fr;

  @media screen and (max-width: 888px) {
    display: flex;
    flex-direction: column;
    align-items: center;
  }

  & .collection-item {
    margin-bottom: 30px;
  }
`;
