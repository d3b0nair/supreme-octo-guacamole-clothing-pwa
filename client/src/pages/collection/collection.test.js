import React from "react";
import { render, cleanup } from "@testing-library/react";
import { CollectionPage } from "./collection.component";

const mockProps = (otherProps) => ({
  ...otherProps,
  collection: { title: "", items: [] },
});
const component = (mockProps) => <CollectionPage {...mockProps} />;

afterEach(cleanup);

describe("Collection component", () => {
  it("should mount", () => {
    const { container } = render(component(mockProps()));
    expect(container).toMatchSnapshot();
  });
});
