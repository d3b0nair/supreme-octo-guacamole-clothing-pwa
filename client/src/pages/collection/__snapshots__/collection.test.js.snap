// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Collection component should mount 1`] = `
<div>
  <div
    class="sc-iCfMLu iRZbeT"
  >
    <h2
      class="sc-furwcr dFENJw"
    />
    <div
      class="sc-pVTFL ehVcDl"
    />
  </div>
</div>
`;
